src/content/en/updates/2020/03/devtools.md

```diff
@@ -0,0 +1,83 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2020-03-09 #}
+{# wf_published_on: 2020-03-09 #}
+{# wf_tags: chrome81, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 82) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Emulate vision deficiencies 
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e847a020093fbd3c367eda02a7008feae5e28641 #}
+
+Send feedback to [Chromium issue #1003700](https://crbug.com/1003700).
+
+## Clear filters button in Network panel {: #TODO }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/2604c1917006056e82007ef892e6223218cbc346 #}
+
+Send feedback to [Chromium issue #1056666](https://crbug.com/1056666).
+
+## Dock-to-left in the Command Menu {: #dock-to-left }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/7ff2e8eda31b86a8e249983f182e20e65c303556 #}
+
+Send feedback to [Chromium issue #1011679](https://crbug.com/1011679).
+
+## COEP {: #COEP }
+
+{# Not sure if COEP is launching behind a flag in Chrome 82. We usually don't cover web platform or DevTools experiments. #}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/a2afe0b0d0b307a2bc2c1a8e9ed5c97f1b89569c #}
+
+Send feedback to [Chromium issue #1051466](https://crbug.com/1051466).
+
+## Lighthouse 5.7.1 {: #lighthouse }
+
+New audits:
+
+* [CLS](https://github.com/GoogleChrome/lighthouse/commit/a8a0f8fd369faaa23b97d8803bfc220db31e4769#diff-eb7e207cabf22c8000b5abe8f18c6daf)
+* [Legacy JS](https://github.com/GoogleChrome/lighthouse/pull/10303)
+* [Maskable icon](https://github.com/GoogleChrome/lighthouse/pull/10370)
+* [Duplicate JS](https://github.com/GoogleChrome/lighthouse/commit/ea9d226606c5b534d4259253baac5f709a9e3d7c#diff-eb7e207cabf22c8000b5abe8f18c6daf)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/933525c2bdcf22d49b5688c04cad7bc30c476b3d #}
+
+Send feedback to [Chromium issue #772558](https://crbug.com/772558).
+
+## Audits panel renamed to Lighthouse {: #lighthouse }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/2ae53ef04d84c4201836eb63238e97bc180ea54d #}
+
+## Clear all Local Overrides in a folder {: #TODO }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0fc949c8b168dea42d12432f4591ee4c5dabb9a5 #}
+
+Send feedback to [Chromium issue #1016501](https://crbug.com/1016501).
+
+## Updated Long tasks UI {: #long-tasks }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e19afdbb2fb24f4123dc7b8b62f38d359ff2d916 #}
+
+Send feedback to [Chromium issue #1054447](https://crbug.com/1054447).
+
+## `$$` now returns elements across shadow DOM boundaries {: #querySelectorAll }
```

Comment:
This should be removed. It doesn’t affect the DevTools Console `$$`, but rather the helper with the same name which we use in the e2e tests. cc @paullewis @TimvdLippe

Reply:
Ah, that makes more sense! I was like "why is Tim talking as if this is a new feature? Pretty sure we've had `$$` forever..."

Reply:
Removed this section